packages/client/modules/outcomeCard/components/OutcomeCardAssignMenu/TaskFooterTeamAssigneeMenu.tsx

```diff
@@ -23,14 +23,19 @@ const TaskFooterTeamAssigneeMenu = (props: Props) => {
   const {userIds, teamIds} = useUserTaskFilters(viewer.id)
   const {team, id: taskId} = task
   const {id: teamId} = team
-
+  const {teams} = viewer
   const assignableTeams = useMemo(() => {
-    const {teams} = viewer
-    const filteredTeams = userIds ? teams.filter(({teamMembers}) =>
-      !!teamMembers.find(({userId}) => userIds.includes(userId))
-    ) : (teamIds ? teams.filter(({id}) => teamIds.includes(id)) : teams)
-    return filteredTeams.filter((team) => team.id !== teamId)
```

Comment:
Also here @tianrunhe & @mattkrick , by filtering the currently assigned team from the dropdown, we get an empty menu if the user is only in one team:

<img width="378" alt="empty-menu" src="https://user-images.githubusercontent.com/39854876/94568474-6c2ff700-0264-11eb-9996-007267946d61.png">

We have an issue open for this [here](https://github.com/ParabolInc/parabol/issues/4129). In this file, I'm including the `teamId` in the menu so it always shows something. If the user clicks the currently assigned team, the mutation doesn't fire.